packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/AssociationEditor.tsx

```diff
@@ -444,7 +445,7 @@ export const AssociationEditor = observer(
       >
         <ResizablePanelGroup orientation="horizontal">
           <ResizablePanel minSize={56}>
-            <div className="panel">
+            <Panel>
               <div className="panel__header">
```

Comment:
We definitely miss a few spots like these here and there, but I think this is a very encouraging first step 🙌

Reply:
😅 yes I will continue!